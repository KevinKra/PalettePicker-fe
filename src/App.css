.App {
  border: 1px solid red;
  min-height: 100vh;
  max-width: 100vw;
}

.NavBar {
  position: relative;
  border: 2px solid green;
  min-height: 5vh;
  z-index: 2;
  background-color: white;
}

.MainPage {
  min-height: 95vh;
}

.ProjectBox {
  border: 1px solid red;
  min-height: 25vh;
}

.Palette {
  border: 1px solid green;
  margin: 1rem;
  height: 100px;
  width: 30%;
}
