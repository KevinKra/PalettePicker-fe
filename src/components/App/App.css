.App {
  position: relative;
  min-height: 100vh;
  max-width: 100vw;
  font-family: "Comfortaa", cursive;
}
