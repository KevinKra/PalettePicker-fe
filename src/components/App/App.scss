.App {
	display: grid;
	grid-template-rows: 7vh 1fr;
	min-height: 100vh;
	font-family: "Comfortaa", cursive;
}
