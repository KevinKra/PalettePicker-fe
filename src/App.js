import React from "react";
import "./App.css";
import ProjectsContainer from "./components/ProjectsContainer/ProjectsContainer";
import PalettePicker from "./components/PalettePicker/PalettePicker";

function App() {
  return (
    <div className="App">
      <header className="NavBar">
        <h2>PalettePicker</h2>
        <form className="navbar-form">
          <button>Login</button>
          <button>Register</button>
        </form>
      </header>
      <main className="MainPage">
        <PalettePicker />
        <ProjectsContainer />
      </main>
    </div>
  );
}

export default App;
