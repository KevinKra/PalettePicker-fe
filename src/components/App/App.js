import React from 'react';
import LandingPage from '../../_screens/LandingPage/LandingPage';
import NavBar from '../../containers/NavBar/NavBar';
import PaletteEditor from '../PaletteEditor/PaletteEditor';
import SavePopup from '../../containers/SavePopup/SavePopup';
import userPopup from '../../containers/userPopup/userPopup';
import { Route } from 'react-router-dom';
import './App.scss';

function App() {
	return (
		<section className="App">
			<Route path="/" component={NavBar} />
			<Route path="/" component={LandingPage} />
			<Route exact path="/edit-palette" component={PaletteEditor} />
			<Route exact path="/save-palette" component={SavePopup} />
			<Route exact path="/login" component={userPopup} />
			<Route exact path="/register" component={userPopup} />
		</section>
	);
}

export default App;
