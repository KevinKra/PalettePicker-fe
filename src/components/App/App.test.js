import React from "react";
import App from "./App";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

describe("App", () => {
  it("should match the snapshot", () => {
    const wrapper = shallow(<App />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
