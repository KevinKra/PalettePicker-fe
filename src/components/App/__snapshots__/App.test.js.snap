// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<section
  className="App"
>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(withRouter(NavBar))",
        "type": [Function],
      }
    }
    path="/"
  />
  <Route
    component={[Function]}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/edit-palette"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(SavePopup)",
        "type": [Function],
      }
    }
    exact={true}
    path="/save-palette"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(userPopup)",
        "type": [Function],
      }
    }
    exact={true}
    path="/login"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(userPopup)",
        "type": [Function],
      }
    }
    exact={true}
    path="/register"
  />
</section>
`;
